kimchi/src/tests/lookup.rs

```diff
@@ -582,3 +582,50 @@ fn test_runtime_table_only_one_table_with_id_zero_with_non_zero_entries_random_v
 
     setup_successfull_runtime_table_test(vec![cfg], vec![runtime_table], lookups);
 }
+
+#[test]
+fn test_lookup_with_initial_xor_dummy_value_when_no_xor_table() {
+    let seed: [u8; 32] = thread_rng().gen();
+    eprintln!("Seed: {:?}", seed);
+    let mut rng = StdRng::from_seed(seed);
+    let max_len: u32 = 100u32;
+    let max_table_id: i32 = 100;
+
+    // No zero-length table
+    let len = 1u32 + rng.gen_range(0u32..max_len);
+    // No table of ID 0
+    let table_id: i32 = 1 + rng.gen_range(0i32..max_table_id);
```

Comment:
Remind me: the `RANGE_CHECK` table will not be included because there is no lookup gate into it? This is why `table_id == 1` will work here and won't collide with `RANGE_CHECK_TABLE_ID`?

Reply:
Correct.